### PR TITLE
this is a JanusGraph Server fat image

### PR DESCRIPTION
**Pull request by goern**

### CODE REVIEW

**Comment by kbsingh** [comment]
Looks like the Dockerfile is not in the root dir, but under janusgraph-openshift/ might need to edit that into the index entry

**Comment by kbsingh** [comment]
@bamachrn @mohammedzee1000 do we not have something in the registry already that satisfies the same piece : https://github.com/goern/janusgraph-openshift/blob/master/images/janusgraph/Dockerfile#L1

**Comment by mohammedzee1000** [comment]
Can you add depends-on: null to the entry

**Comment by bamachrn** [comment]
> do we not have something in the registry already that satisfies the same piece : https://github.com/goern/janusgraph-openshift/blob/master/images/janusgraph/Dockerfile#L1

Not yet, @mohammedzee1000 lets get all the language pieces in.

**Comment by bamachrn** [comment]
> Can you add depends-on: null to the entry

 * File Name : janusgraph.yml
   * Entry ID : 36dec1b0f748e722d7e98eb03ed16aa15f78fdfd9b0e1bd848c3cd5bc6aba0da
   * Entry    : {'app-id': 'janusgraph', 'git-url': 'https://github.com/goern/janusgraph-openshift', 'git-path': 'images/janusgraph/', 'git-branch': 'master', 'notify-email': 'goern@redhat.com', 'desired-tag': 'latest', 'target-file': 'Dockerfile', 'build-context': './', 'job-id': 'server', 'id': 1}
   * Valid    : NO
   * E Missing depends-on

**Comment by mohammedzee1000** [comment]
@goern ^

File: index.d/janusgraph.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    app-id: janusgraph
+    job-id: server
+    git-url: https://github.com/goern/janusgraph-openshift
+    git-branch: master
+    git-path: images/janusgraph/
+    target-file: Dockerfile
+    desired-tag: latest
+    notify-email: goern@redhat.com
+    build-context: ./
+    depende-on: null
```
Comment (mohammedzee1000) [inline index.d/janusgraph.yml]: Typo, its depends-on.

**Comment by mohammedzee1000** [comment]
Ok, you also need to add a cccp.yml file in your repository https://github.com/goern/janusgraph-openshift/tree/master/images/janusgraph with atleast one key job-id: <sameidasgivenhere>

**Comment by goern** [comment]
ping @mohammedzee1000 thx!

**Comment by mohammedzee1000** [comment]
@goern The file must be present at the git-path, alongside the target-file.

**Comment by goern** [comment]
@mohammedzee1000 could you rekick the job?

**Comment by dharmit** [comment]
ok to test

**Comment by mohammedzee1000** [comment]
ok to test

**Comment by goern** [comment]
after this merge, I cant see it on https://ci.centos.org/ How do I get the image to https://registry.centos.org/containers/ ?

**Comment by mohammedzee1000** [comment]
@goern  Please check https://registry.centos.org/pipeline-logs/ODU4ZDdlMjJiMm/build_logs.txt

**Comment by goern** [comment]
duh... how could I get notifications for that? or did i miss it somehow?

**Comment by bamachrn** [comment]
@goern build system sent you emails on `goern@redhat.com`. I believe you missed it. also we tried reaching  out on the same email id, informing you the build failures. would you please check for the emails? so that we can also confirm, if  there is issue with notification system.

**Comment by goern** [comment]
email is good, maybe google sorted it to some mysterious place...

**Pull request merged**